experiments/n2v/bsd68/careamics/careamics-bsd68.py

```diff
@@ -48,20 +47,10 @@
     num_epochs=50,
     masked_pixel_percentage=0.2,
     struct_n2v_axis="none",
-    model_params={
```

Comment:
So `model_params` was correct, just the other two parameters were not existing (yet).

Now they are in CAREamics (since this morning): https://github.com/CAREamics/careamics/pull/483

Make sure that the optimizer and lr_scheduler are indeed the same as in the tensorflow implementation!